docs/getting_started.md

```diff
@@ -521,42 +519,46 @@ Generate a random, ten-byte piece of data:
     /* Clean up */
     mbedtls_psa_crypto_free();
 ```
+To generate a random key, use `psa_generate_key()` instead of `psa_generate_random()`.
```

Comment:
This is better placed at the beginning of this section, perhaps as a note or warning. Readers are likely to miss this if it is at the bottom and they followed along until they got what they thought they needed and stopped reading.